src/rete/nodes/EditorGainNode.tsx

```diff
@@ -5,33 +5,32 @@ import { LabeledInputControl } from '../controls/LabeledInputControl'
 export class EditorGainNode extends Classic.Node<
     {
         signal: Classic.Socket
-        baseGain: Classic.Socket
         additionalGain: Classic.Socket
     },
     { signal: Classic.Socket },
-    {}
+    { baseGain: Classic.InputControl<'number', number> }
 > {
     width = 180
     height = 200
```

Comment:
Let's update the height of the node now that the socket's been removed so that it looks right

Reply:
Do you have a specific height in mind, or whatever value looks best?